browser/js/main.js

```diff
@@ -43,8 +155,8 @@
           response.text().then(catchError);
         }
       })
-      .catch(catchError)
-    })
+      .catch(catchError);
+    });
```

Comment:
Keeping with the style of the hole codebase. I should add something to enforce it, because I keep forgetting to add them :/